ndc-models/src/lib.rs

```diff
@@ -205,6 +205,9 @@ pub struct ObjectField {
     /// The type of this field
     #[serde(rename = "type")]
     pub r#type: Type,
+    /// The arguments available to the field - Matches implementation from CollectionInfo
+    #[serde(skip_serializing_if = "Option::is_none")]
```

Comment:
You already have `skip_serializing_none` on the `struct`.

Reply:
great!